packages/gatsby-theme-styleguide/src/gatsby-components/Code/normalize.js

```diff
@@ -0,0 +1,155 @@
+const COMMENT_START = new RegExp(`(#|\\/\\/|\\{\\/\\*|\\/\\*+|<!--)`)
```

Comment:
use a link to tell where does it come from

Reply:
Added some comment to explain, what and where it comes from